src/links/v1/providers.json

```diff
@@ -52,6 +52,17 @@
         "link_type": "external"
       }
     },
+    {
+      "type": "links",
+      "id": "cmr",
+      "attributes": {
+        "name": "CMR Database",
+        "description": "Computational materials repository.",
+        "base_url": "https://cmr-optimade.fysik.dtu.dk",
```

Comment:
This URL should actually be the URL of an "index meta-database"; we typically host that for you if you are only serving one database, but for CMR it might make sense to set up a custom index over all of your datasets. There are some examples in the index meta-db folder of this repo.

Would it be easier to have a quick video call about this sometime?

Reply:
That would be great.  How about tomorrow at 10:00, danish time?